src/record/import/utils/error.ts

```diff
@@ -1,113 +1,112 @@
+        const bulkRequestIndex = this.error.bulkRequestIndex ?? 0;
+        const indexMatch = key.match(/records\[(?<index>\d+)\]/);
+        const index =
+          Number(indexMatch?.groups?.index) +
+          bulkRequestIndex * this.chunkSize +
+          this.offset;
 
-    for (const [key, value] of orderedErrors) {
-      const bulkRequestIndex = e.bulkRequestIndex ?? 0;
-      const indexMatch = key.match(/records\[(?<index>\d+)\]/);
-      const index =
-        Number(indexMatch?.groups?.index) +
-        bulkRequestIndex * chunkSize +
-        offset;
+        const formatInfo = this.records[index].metadata.format;
+        const fieldCode = this._getFieldCodeByErrorKeyWithSchema(
+          key,
+          this.recordSchema
+        );
+        if (formatInfo.firstRowIndex === formatInfo.lastRowIndex) {
+          errorMessage += `  An error occurred on ${fieldCode} at row ${
+            formatInfo.lastRowIndex + 1
+          }.\n`;
+        } else {
+          errorMessage += `  An error occurred on ${fieldCode} at rows from ${
+            formatInfo.firstRowIndex + 1
+          } to ${formatInfo.lastRowIndex + 1}.\n`;
+        }
 
-      const formatInfo = records[index].metadata.format;
-      const fieldCode = getFieldCodeByErrorKeyWithSchema(key, recordSchema);
-      if (formatInfo.firstRowIndex === formatInfo.lastRowIndex) {
-        errorMessage += `  An error occurred on ${fieldCode} at row ${
-          formatInfo.lastRowIndex + 1
-        }.\n`;
-      } else {
-        errorMessage += `  An error occurred on ${fieldCode} at rows from ${
-          formatInfo.firstRowIndex + 1
-        } to ${formatInfo.lastRowIndex + 1}.\n`;
-      }
-
-      for (const message of value.messages) {
-        errorMessage += `    Cause: ${message}\n`;
+        for (const message of value.messages) {
+          errorMessage += `    Cause: ${message}\n`;
+        }
       }
     }
+    return errorMessage;
   }
-  return errorMessage;
-};
-
-/**
- * Parse error key to string array. For example:
- * Input: records[0].Table.value[1].value.table_text_0.value
- * Output: ['records', '0', 'Table', 'value', '1', 'value', 'table_text_0', 'value']
- * @param {string} errorKey
- */
-const parseErrorKey = (errorKey: string): string[] => {
-  return errorKey.split(/[[\].]+/);
-};
-
-const getFieldCodeByErrorKeyWithSchema = (
-  errorKey: string,
-  schema: RecordSchema
-) => {
-  const parsedErrorKey = parseErrorKey(errorKey);
-  const fieldCodeIndex = 2;
-  const fieldCode = parsedErrorKey[fieldCodeIndex];
 
-  const field = schema.fields.find((f) => f.code === fieldCode);
-  if (field === undefined) {
-    return fieldCode;
+  /**
+   * Parse error key to string array. For example:
+   * Input: records[0].Table.value[1].value.table_text_0.value
+   * Output: ['records', '0', 'Table', 'value', '1', 'value', 'table_text_0', 'value']
+   * @param {string} errorKey
+   */
+  _parseErrorKey(errorKey: string): string[] {
```

Comment:
Use the `private` method explicitly.

```suggestion
  private _parseErrorKey(errorKey: string): string[] {
```